WordPress/src/main/java/org/wordpress/android/ui/reader/viewmodels/ReaderPostListViewModel.kt

```diff
@@ -281,6 +284,11 @@ class ReaderPostListViewModel @Inject constructor(
         _filtersMatchCount.postValue(currentValue)
     }
 
+    fun onBottomSheetActionClicked(selectedTabIndex: Int) {
+        _isBottomSheetShowing.postValue(Event(false))
```

Comment:
This might be a personal preference, but "isBottomSheetShowing" evokes me that the LiveData hold the current state. However, it seems we are sending an Event there, so they are not holding the current state, but they emit an event when we want to show/hide the bottom sheet.
I'd consider renaming it to something like `changeBottomSheetVisibility` or similar. Wdyt?

Reply:
👍Changes here 19fb6dd30d5831e4162b311fbf9d6680913fbf87